api/v1beta1/openstackbaremetalset.go

```diff
@@ -97,7 +98,14 @@ func VerifyBaremetalSetScaleUp(
 	if newBmhsNeededCount > 0 {
 		// We have new BaremetalHosts requested, so search for BaremetalHosts that don't have consumerRef or online set
 
-		l.Info("Attempting to find BaremetalHosts for scale-up of OsBaremetalSet", "OsBaremetalSet", instance.Name, "quantity", newBmhsNeededCount)
+		labelStr := ""
+
+		if len(instance.Spec.BmhLabelSelector) > 0 {
+			labelStr = fmt.Sprintf("%v", instance.Spec.BmhLabelSelector)
+			labelStr = strings.Replace(labelStr, "map[", "[", 1)
+		}
+
+		l.Info("Attempting to find BaremetalHosts for scale-up of OpenStackBaremetalSet", "OpenStackBaremetalSet", instance.Name, "quantity", newBmhsNeededCount, "labels", labelStr)
```

Comment:
We probably should add the namespace if we're adding the label, if the user would not know the spec they have used with namespace/label.

Reply:
Ack, added